.eslintrc

```diff
@@ -120,6 +120,10 @@
                 "jest/no-focused-tests": "error",
                 "jest/valid-expect": "error",
                 "jest/valid-expect-in-promise": "error"
+            },
+
+            "globals": {
+                "BigInt": true
```

Comment:
Do we still need this?

Reply:
Yes, because instead of using the literal, we're using the BigInt function.

Reply:
I am surprised that it's not already a global defined in ESLint. This looks fishy.

Reply:
Instead of adding `BigInt`, I would recommend enabling [newly added globals](https://github.com/eslint/eslint/pull/13356#issuecomment-633686554) using `env`:

```json
"env": {
    "es6": true,
    "es2017": true,
    "es2020": true,
}
```